src/plugins/intel_npu/src/plugin/npuw/partitioning/patterns/opt.cpp

```diff
@@ -1653,6 +1672,63 @@ HostGatherDQ::HostGatherDQ(Context::Ref ctx) {
     register_matcher(std::make_shared<opp::Matcher>(qmul, "HostGatherDQ"), std::move(callback));
 }
 
+HostGatherNF4::HostGatherNF4(Context::Ref ctx) {
+    auto pids = opp::wrap_type<ov::op::v0::Parameter>();
+    auto cvtids = opp::wrap_type<ov::op::v0::Convert>({pids});
+
+    auto qweight = opp::wrap_type<ov::op::v0::Parameter>();
+    auto qcvtw = opp::wrap_type<ov::op::v0::Convert>({qweight});
+    auto qweightt = opp::wrap_type<ov::op::v0::Constant>();
```

Comment:
if you use `qweightt` with extra `t` to denote a table, please rename it to `qtable` - as now it looks like a typo given there's also `qweight` few lines above (and both are used in the callback)

Reply:
Done